packages/pn-commons/src/components/CodeModal/CodeModal.tsx

```diff
@@ -147,6 +147,7 @@ const CodeModal = memo(
               onClick={cancelCallback}
               fullWidth={isMobile}
               data-testid="codeCancelButton"
+              sx={{ marginTop: isMobile ? "16px" : 0 }}
```

Comment:
@NicolaGiornetta I replaced the marginTop , it was 10px, I set 16px, to keep uniform separation between buttons in dialogs in mobile view. Please confirm this adjustement in order to close the PR, thanks!